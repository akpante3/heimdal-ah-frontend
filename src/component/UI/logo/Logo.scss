.logo {
  width: 10% x;
  height: 70px;
}