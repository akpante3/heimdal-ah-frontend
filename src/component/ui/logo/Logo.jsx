import React from 'react';
import { Link } from 'react-router-dom';
import './Logo.scss';

const Logo = () => (
  <Link to='/'>
    <img className='logo' src="./src/images/logo.png" alt="heimdal-logo" />
  </Link>
);

export default Logo;
