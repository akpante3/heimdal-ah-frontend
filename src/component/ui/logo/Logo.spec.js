/* eslint-disable no-undef */
import React from 'react';
import toJson from 'enzyme-to-json';
import Logo from './Logo';


describe('testing the logo', () => {
  it('should render the image', () => {
    const wrapper = shallow(<Logo />);
    const image = wrapper.find('img');
    expect(image.length).toBe(1);
  });

  it('should create a snapshot of the logo', () => {
    const tree = shallow(<Logo />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
