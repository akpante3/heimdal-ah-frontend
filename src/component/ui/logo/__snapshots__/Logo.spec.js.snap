// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing the logo should create a snapshot of the logo 1`] = `
<Link
  replace={false}
  to="/"
>
  <img
    alt="heimdal-logo"
    className="logo"
    src="./src/images/logo.png"
  />
</Link>
`;
